django-stubs/contrib/admin/options.pyi

```diff
@@ -199,7 +199,7 @@ class ModelAdmin(BaseModelAdmin):
     ) -> None: ...
     def save_form(self, request: Any, form: Any, change: Any): ...
     def save_model(self, request: Any, obj: Any, form: Any, change: Any) -> None: ...
```

Comment:
Do we need to type `obj` as `_T` here?

Reply:
sure do! i was really tired when i worked on this, sorry :p

Reply:
No worries! Thanks a lot for your help!